chapters/ch13.md

```diff
@@ -1,40 +1,26 @@
 # 13. You want to build a new email anti-spam system. Your team has several ideas:
-->
-
+-> Bạn mong muốn xây dựng một hệ thống phòng chống email rác mới. Đội của bạn có một vài ý tưởng:
 
 * Collect a huge training set of spam email. For example, set up a “honeypot”: deliberately send fake email addresses to known spammers, so that you can automatically harvest the spam messages they send to those addresses.
-->
-
+-> Thu thập một tập huấn luyện lớn về email rác. Ví dụ như thiết lập một Honeypot: cố ý gửi các địa chỉ email giả đến những spammer đã biết, và bạn có thể thu thập các tin nhắn rác mà họ gửi đến địa chỉ đó một cách tự động.
 
 * Develop features for understanding the text content of the email.
-->
-
-
-* Develop features for understanding the email envelope/header features to show what set
-->
-
-
-of internet servers the message went through.
-->
+-> Phát triển những tính năng để hiểu được nội dung bằng văn bản trong email.
 
+* Develop features for understanding the email envelope/header features to show what set of internet servers the message went through.
+-> Phát triển những tính năng để hiểu được các đặc tính của phông thư/nhãn thư từ email nhằm hiển thị tập hợp các máy chủ internet mà thư đã đi qua.
 
 * and more.
-->
-
+->và nhiều hơn thế.
 
 Even though I have worked extensively on anti-spam, I would still have a hard time picking one of these directions. It is even harder if you are not an expert in the application area.
-->
-
+-> Mặc dù tôi đã kinh qua rất nhiều trong việc phòng chống email rác, tôi vẫn có một khoảng thời gian khó khăn khi lựa chọn hướng tiếp cận này. Điều này sẽ còn khó hơn nếu bạn không phải là một chuyên gia trong việc ứng dụng lĩnh vực này.
 
 So don’t start off trying to design and build the perfect system. Instead, build and train a basic system quickly—perhaps in just a few days [5]. Even if the basic system is far from the “best” system you can build, it is valuable to examine how the basic system functions: you will quickly find clues that show you the most promising directions in which to invest your time. These next few chapters will show you how to read these clues.
-->
-
+-> Vì vậy, bạn không nên khởi động bằng việc cố gắng thiết kế và xây dựng một hệ thống hoàn hảo. Thay vào đó, hãy xây dựng và huấn luyện nhanh một hệ thống cơ bản- có lẽ sẽ mất một vài ngày.[5] Ngay cả khi hệ thống cơ bản trông khác xa với hệ thống tốt nhất mà bạn có thể xây dựng, bạn vẫn có thể kiểm tra cách thức hoạt động của hệ thống cơ bản này: bạn sẽ nhanh chóng tìm ra một vài đầu mối (ý tưởng) chỉ cho bạn những hướng đi đầy hứa hẹn nhất để đầu tư thời gian của bạn. Trong những chương tiếp theo sẽ chỉ cho bạn cách đọc những đầu mối (thông tin) này.
```

Comment:
- "perhaps in just a few days" = "có thể là/chỉ trong vài ngày" (thôi) (ý ở đây là bạn đừng vội tìm cách làm ra một hệ thống thật hoàn hảo, thay vào đó hãy nhanh chóng tạo ra một hệ thống "chạy được" đã, đừng tốn nhiều hơn vài ngày vào việc setup  một hệ thống basic, functional.

- "trông khác xa" --> khác xa

- "it is valuable to examine how the basic system functions" = việc kiểm tra một hệ thống cơ bản làm việc như thế nào *vẫn rất ý nghĩa* (mình thấy bạn bỏ mất cái ý valuable)

- "đầy hứa hẹn nhất" --> "hứa hẹn nhất"